data/mods/Magiclysm/worldgen/elf_workshop.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "type": "palette",
+    "id": "elf_palette",
+    "terrain": { "3": "t_rdoor_c", "1": "t_strconc_wall", ".": "t_metal_floor_olight" },
```

Comment:
`t_metal_floor_olight` every floor tile will have a light in it. We generally reserve overhead lights for labs, Placing some Magiclsym everlasting torches will be better. `t_metal_floor` is the normal metal floor.